includes/class-wc-stripe-webhook-handler.php

```
@@ -1487,7 +1550,7 @@ public function process_checkout_session( object $notification ): void {
 		 */
 		$allowed_payment_processing_statuses = apply_filters(
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $allowed_payment_processing_statuses. Keep variable name length under 20. (undefined)

(LongVariable)